docs/src/rsh/consensus/index.md

```diff
@@ -644,7 +644,7 @@ load: /examples/dominant-assurance/index.rsh
 range: 143 - 143
 ```
 
-To check whether the address is in the set, `{!rsh} s.member(ADDRESS)` is used.
+To check whether the `{!rsh} ADDRESS` is in the set, `{!rsh} s.member(ADDRESS)` is used.
```

Comment:
I actually like the original, because we're talking about the "idea" of an address. I do think the phrasing should be "an address"